scripts/west_commands/ncs-provision.py

```diff
@@ -38,6 +39,15 @@ def do_add_parser(self, parser_adder):
                                    choices=["nrf54l15"], required=True)
         upload_parser.add_argument("--dev-id", help="Device serial number")
 
+        revoke_parser = subparsers.add_parser("revoke", help="Disable key")
+        revoke_parser.add_argument(
+            "-i", "--keyid", type=int, choices=[0, 1, 2], required=True,
+            help="Input .pem file with ED25519 private key"
```

Comment:
need relevant help message, it should explains how the parameter values maps to subsequent keys provisioned on the device:  0 for the 1st one and so one.

Reply:
tbh this is not very future proof. Keyslots should be parsed per SOC.